test_utils/src/lib.rs

```diff
@@ -223,3 +223,208 @@ macro_rules! assert_error {
         assert!(format!("{:?}", $result).contains(&format!($($error),+)));
     }
 }
+
+#[macro_export]
+macro_rules! test_interpreter_and_jit {
+    (override_budget => $override_budget:expr, $executable:expr, $mem:tt, $context_object:expr $(,)?) => {{
+        let expected_instruction_count = $context_object.get_remaining();
+        #[allow(unused_mut)]
+        let mut context_object = $context_object;
+        if !$override_budget {
+            const INSTRUCTION_METER_BUDGET: u64 = 1024;
+            context_object.remaining = INSTRUCTION_METER_BUDGET;
+        }
```

Comment:
override_budget == true will NOT override the budget.
override_budget == false WILL override the budget.

Reply:
Ah yes, we should invert the flag and its use sites.